build.fsx

```diff
@@ -61,7 +61,8 @@ Target "Build" (fun _ ->
             (fun p -> 
                 { p with
                     Project = project
-                    Configuration = configuration })
+                    Configuration = configuration
+                    AdditionalArgs = ["--no-incremental"]}) // i.e. "Rebuild"
```

Comment:
Additional change - we should be calling "Rebuild" to be consistent with previous build script.